packages/cli/src/__tests__/commands/Generate.test.ts

```diff
@@ -82,6 +82,189 @@ describe('using cli', () => {
     `)
   }, 60_000) // timeout
 
+  it('should display the right yarn command for custom outputs', async () => {
+    ctx.fixture('custom-output-yarn')
+    const data = await ctx.cli('generate')
+
+    if (typeof data.signal === 'number' && data.signal !== 0) {
+      throw new Error(data.stderr + data.stdout)
+    }
+
+    if (getClientEngineType() === ClientEngineType.Library) {
+      expect(data.stdout).toMatchInlineSnapshot(`
+        Prisma schema loaded from prisma/schema.prisma
+
+        ✔ Generated Prisma Client (v0.0.0) to ./generated/client in XXXms
+
+        Prisma Client has been generated to a custom path:
+
+        1. Make it a dependency of your project
+        \`\`\`
+        # adapt this relative path if needed
+        yarn add db@link:./generated/client
+        \`\`\`
+
+        More information: https://pris.ly/d/custom-output
+
+        2. Start using Prisma Client
+        \`\`\`
+        import { PrismaClient } from 'db'
+        const prisma = new PrismaClient()
+        \`\`\`
+
+        More information: https://pris.ly/d/client
+
+        ┌─────────────────────────────────────────────────────────────┐
+        │  Deploying your app to serverless or edge functions?        │
+        │  Try Prisma Accelerate for connection pooling and caching.  │
+        │  https://pris.ly/cli/accelerate                             │
+        └─────────────────────────────────────────────────────────────┘
+
+      `)
+    }
+  })
+
+  it('should display the right npm command for custom outputs', async () => {
+    ctx.fixture('custom-output-npm')
+    const data = await ctx.cli('generate')
+
+    if (typeof data.signal === 'number' && data.signal !== 0) {
+      throw new Error(data.stderr + data.stdout)
+    }
+
+    if (getClientEngineType() === ClientEngineType.Library) {
+      expect(data.stdout).toMatchInlineSnapshot(`
+        Prisma schema loaded from prisma/schema.prisma
+
+        ✔ Generated Prisma Client (v0.0.0) to ./generated/client in XXXms
+
+        Prisma Client has been generated to a custom path:
+
+        1. Make it a dependency of your project
+        \`\`\`
+        # adapt this relative path if needed
+        npm i db@./generated/client
```

Comment:
@millsp I guess you tried this command already?

Reply:
Yes